src/pages/App.js

```diff
@@ -430,24 +423,6 @@ class App extends Component {
     this.setState({ showWeb3ErrorBanner: false })
   }
 
-  static getDerivedStateFromError(_) {
-    return { didError: true }
-  }
-
-  componentDidCatch(error, errorInfo) {
-    try {
-      Sentry.withScope(scope => {
-        scope.setExtras(errorInfo)
-        Sentry.captureException(error)
-      })
-    } catch (error) {
-      console.error(error.message)
-    }
-    // Dump the persistent cache on error, just in case that is causing some
-    // un-recoverable state.
-    clearAll()
```

Comment:
We also aren't clearing state anymore, is that important?

Reply:
Both of these are great callouts. What I'm hoping to do is just be consistent with both flows, so do we think it makes sense to clear cache on a root saga error as well kinda thing?

Reply:
I think so, if something is erroring in a saga it could be cache related

Reply:
Wait, is the persistent cache even being used? @piazzatron

Reply:
persistent cache is not being used